packages/app-page-builder-elements/src/modifiers/styles/text.ts

```diff
@@ -17,7 +18,10 @@ const text: ElementStylesModifier = ({ element, theme }) => {
 
         let breakpointStyles: CSSObject = {};
 
-        const typographyStyles = theme?.styles?.typography?.[values.typography];
+        const typographyStyles = getTypographyStyleById(
```

Comment:
yes, we need the typography style object by id

Reply:
It might not be a bad idea to simply id `stylesById` on the `typography` level in the theme object, but that's for another day. Gonna mark this one as resolvled.